index.html

```diff
@@ -1,51 +1,77 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
-   
 
-<div id="container">
 
 
 
-<img id="mylogo" src="/Portfolio-setup-and-mobile-version-skeleton/Image/Logo david.jpg" alt="">
-<div class="Header">
+  <section class="top-nav">
+    <div>
+      <img id="mylogo" src="/Image/logo-david-nuevo-2021.png" alt="">
+    </div>
+    <label for="menu-toggle">MY FIRST WEB PAGE</label>
+    <input id="menu-toggle" type="checkbox" />
+    <label class='menu-button-container' for="menu-toggle">
+      <span class='menu-button'></span>
+    </label>
+    <ul class="menu">
+      <li>Portfolio</li>
+      <li>About</li>
+      <li>Contact</li>
+
+
+    </ul>
+  
+  </section>
+  <h1> </h1>
+  <img id="img11" src="/Image/Rectangle6.svg" alt="">
+  <img id="img1" src="/Image/Ellipse11.svg" alt="">
+  <img id="img2" src="/Image/Ellipse5.svg" alt="">
+  <img id="img3" src="/Image/Ellipse8.svg" alt="">
+  <img id="img4" src="/Image/Ellipse3.svg" alt="">
+  <img id="img5" src="/Image/Rectangle7.svg" alt="">
+  <img id="img6" src="/Image/Rectangle4.svg" alt="">
+  <img id="img7" src="/Image/Ellipse8.svg" alt="">
+  <img id="img8" src="/Image/Ellipse8.svg" alt="">
+  <img id="img9" src="/Image/Rectangle7.svg" alt="">
+  <img id="img0" src="/Image/Ellipse3.svg" alt="">
 
-</div>
-    <nav>
-<ul>
-   <li>Portfolio</li>
-   <li>About</li>
-   <li>Contact</li>
-</ul>
-</nav>
-</div>
 
-<pre id="name">I'm David Verde   
-    Glad to see you! 
+  <pre id="name">I'm David Verde! 
+Glad to see 
+you! 
```

Comment:
As this is the main heading for your Portfolio. this should be written in `<h1>` tag Kindly replace `<pre>` with `<h1>` and there should be only one `<h1>` tag per HTML document. 👍🏻